aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -921,9 +926,9 @@ std::tuple<Tensor, Tensor, Tensor> _lu_with_info_cuda(const Tensor& self, bool p
   }
   if (check_errors) {
     if (self.dim() == 2) {
-      singleCheckErrors(infos_tensor.item<int64_t>(), "lu");
+      singleCheckErrors(infos_tensor.item<int64_t>(), "lu", /*allow_singular=*/true);
```

Comment:
Same comment as earlier.

Reply:
Hopefully, the same answer will be satisfactory.